crates/forge/src/scarb.rs

```diff
@@ -38,9 +41,28 @@ pub struct StarknetContractArtifacts {
     pub casm: String,
 }
 
+#[derive(Debug, PartialEq, Default)]
+pub struct ForgeConfig {
+    /// Should runner exit after first failed test
+    pub exit_first: bool,
+    /// How many runs should fuzzer execute
+    pub fuzzer_runs: Option<u32>,
+    /// Seed to be used by fuzzer
+    pub fuzzer_seed: Option<u64>,
+
+    pub fork: Vec<ForkTarget>,
+}
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct ForkTarget {
+    pub name: String,
+    pub url: Url,
+    pub block_id: BlockId,
+}
+
 /// Represents forge config deserialized from Scarb.toml
 #[derive(Deserialize, Debug, PartialEq, Default)]
-pub struct ForgeConfig {
+pub struct RawForgeConfig {
```

Comment:
Do then need to still be public? Also documentation is no longer valid for these, should be updated.